config.yml.sample

```diff
@@ -1,5 +1,6 @@
 development:
   cookie_secret: sample
+  cookie_ssl_only: false
```

Comment:
Ok, great. Then I think leave this as it is and in the README you can list this as a reason someone might want to do manual provisioning.